pkg/packagekit/updates.less

```diff
@@ -12,14 +12,14 @@ table.listing-ct thead th {
 }
 
 tr.listing-ct-item {
-    vertical-align: baseline;
-
-    .listing-ct-toggle {
-        vertical-align: middle;
-    }
     td {
+        vertical-align: baseline;
         padding-left: 0;
 
+        &.listing-ct-toggle {
+            position: relative;
+            top: 2px;
```

Comment:
Shifting the icon 2 pixels down is due to the border (1px) and the slightly different baselines of the text font and the icon font (approx 1px).